github-status/handler.go

```diff
@@ -119,21 +119,23 @@ func getLogs(status *sdk.CommitStatus, event *sdk.Event) (string, error) {
 }
 
 func reportToGithub(commitStatus *sdk.CommitStatus, event *sdk.Event) error {
+	appID := os.Getenv("github_app_id")
 	if os.Getenv("use_checks") == "false" {
-		return reportStatus(commitStatus.Status, commitStatus.Description, commitStatus.Context, event)
```

Comment:
What kind of value was in `commitStatus.Context` before vs the static GitHub app ID?

Reply:
https://developer.github.com/v3/repos/statuses/

```
context | string | A string label to differentiate this status from the status of other systems. Default: default
```

`default` was always the value

Reply:
OK, so did you pick the GitHub app ID because it should be unique between installations? It could be any identifier as long as it is consistent?

Reply:
Yeh, and its already included in the image as an env var